articles/from-enthusiasm-to-commercial-success.md

```diff
@@ -0,0 +1,74 @@
+happiness and can significantly improve quality of your work.
+
+You should not postpone the start of your journey until "the time is right". The
+time will never be right. As a matter of fact, it will never be "more right"
+than now. So, if you want to make something big, the sooner you start, the
+better.
+
+What you should also have in mind is that it is not only about putting enough
+effort. You need to thoroughly divide and plan your work by knowing your
+priorities. You can base your decisions on something like 80/20 rule. If you
+plan and prioritize your work properly, 20 percent of your work will give you 80
+percent of the value. Of course, these numbers are not exact, but can show that
+majority of the value is delivered by first minority of effort you put into
+something.
+
+No matter in which area you are working, you should know who your main target
+group is. Imagine you created a teleportation machine. If your product can only
+transport individual particles of dust, no one is going to be interested in
+that. Additionally, the outcome of your work must be beneficial for majority of
+your target audience. So, if your teleportation machine could transport humans,
+but only from your living room to your kitchen, you would impress many people,
+but nobody would use your masterpiece.
+
+One of the things that is hardest for accepting is the fact that you should be
+prepared for failure. All the points mentioned before will help you. But,
+whatever you do, there is no silver bullet for succeeding in it. You will just
+have to repeat a try-fail sequence until you finally succeed.
+
+All these points are applicable when you are creating a product. But, would they
+be applicable when you are exploring a new area, let's say, a new technology?
+The answer is simple. If you cannot show the result of your work, you will not
+convince anyone that you are fluent in the area you explored. To do that, you
+need to create something that gets people's attention and that shows benefits of
+your work.
+
+I would like to make an example by sharing my personal success story. In Zuhlke,
+we have focus groups. These are groups of people that are interested in some
+area and are putting an effort into exploring it. Some time ago, we started a
+focus group which explores augmented and virtual reality. We did some
+investigation and got the needed knowledge, but we realized that this is not
+enough. To show other people that we can do it, we decided to create an app that
+would launch us into sight. We analysed the market and noticed there are many
+different shared whiteboard applications, each of them lacking tactile feeling.
+So, that was our goal. The first version we implemented was not even close to
+final. But it was enough to show that it can be done and that we know how to do
+it. Eventually, we were heard, and, after some time, we got the commercial
+project. Also, the app is on its way of becoming our official product.

```

Comment:
"Also, the app is on its way of becoming our official product." -> the meaning an relevance of 'our official product' is unclear . As Zuhlke is not a product company (which not all readers might know), what does it even mean? Can you rewrite this sentence to make a clearer point. Or leave the sentence completely as this is not a very important information in the context of this article.